eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -456,4 +456,38 @@ public function updateImagePath(int $fieldId, string $oldPath, string $newPath):
             ->execute()
         ;
     }
+
+    /**
+     * @throws \eZ\Publish\Core\IO\Exception\InvalidBinaryFileIdException
+     * @throws \Doctrine\DBAL\Driver\Exception
+     * @throws \Doctrine\DBAL\Exception
+     */
+    public function getImageReference(string $uri, int $fieldId): bool
```

Comment:
naming: you're saying `get` image reference, yet your returning bool. It seems you mean `hasImageReference`, right?